app/views/admin/enterprises/form/_users.html.haml

```diff
@@ -14,20 +14,21 @@
     - else
       = owner_email
 
-.row
-  .three.columns.alpha
-    =f.label :user_ids, t('.notifications')
-    - if full_permissions
-      %span.required *
-    %div{'ofn-with-tip' => t('.contact_tip')}
-      %a= t('admin.whats_this')
-  .eight.columns.omega
-    - if full_permissions
-      %select.select2.fullwidth{id: 'receives_notifications_dropdown', name: 'receives_notifications', ng: {model: 'receivesNotifications', init: "receivesNotifications = '#{@enterprise.contact.id}'"}}
-        %option{ng: {repeat: 'user in Enterprise.users', selected: "user.id == #{@enterprise.contact.id}", hide: '!user.confirmed'}, value: '{{user.id}}'}
-          {{user.email}}
-    - else
-      = @enterprise.contact.email
+- if @enterprise.is_distributor
```

Comment:
Maybe ` unless @enterprise.producer_profile_only` would more explicitly restrict it to profile-only enteprises?

Reply:
Yes ` unless @enterprise.producer_profile_only` sounds better and I was going to change it there but I think I spotted a bug.

I don't think `producer_profile_only` ever get's set to `true` anymore, see https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/assets/javascripts/admin/enterprises/controllers/index_producer_panel_controller.js.coffee#L12 it's assigning with `=` instead of comparing with `==` That makes me wonder if the `producer_profile_only` column/method can be deprecated. What do you think?

Reply:
That's a great catch @cillian, I'm not sure of the consequences, tbh.

Even before this PR, I'm not sure how to set the `producer_profile_only` to `true`. This should correspond to set an enterprise as a "Producer" and "Profile only" (i.e., not a Shop, and not a Hub), right?

I've found this spec, https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/controllers/admin/enterprises_controller_spec.rb#L385 :

```

      context "setting producer_profile_only" do
        it "is ignored" do
          spree_post :register, id: enterprise, sells: 'none', producer_profile_only: true
          expect(response).to redirect_to spree.admin_dashboard_path
          expect(enterprise.reload.producer_profile_only).to be false
        end
      end
```

So, if I  understand this correctly, it seems to ignores this setting and as such, `enterprise.producer_profile_only` seems never to be `true`...